src/AppBundle/Entity/Permissions.php

```diff
@@ -31,7 +31,7 @@ public function getPermissionId()
     /**
      * @var \AppBundle\Entity\FennecUser
      *
-     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\FennecUser", cascade={"persist"})
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\FennecUser", cascade={"persist"}, inversedBy="permissions")
```

Comment:
Wird hier nicht ein Mapping von der Nutzer-Datenbank in die Daten-Datenbank erzeugt? Die darf nur entgegengesetzt existieren (von allen Daten-Datenbanken in die eine Nutzer-Datenbank), aber nicht in diese Richtung, weil sonst immer nur in eine Daten-Datenbank, aber nicht in mehrere, gemappt wird.

Reply:
Das stimmt. Gut beobachtet :+1:

Reply:
ich bin nicht sicher ob der Fix reicht - User hat ja noch die Property Permissions, oder? Bitte noch mal allgemein durch gucken, ich glaube das sitzt an mehreren Stellen.

Reply:
Wir sollten vor dem merge nochmal generell über Daten-Datenbanken, deren Sinn, Implementierung und Notwendigkeit sprechen.

Reply:
Der FennecUser hatte tatsächlich noch die permission property, habe die jetzt rausgenommen. Im Moment linkt WebuserData auf FennecUser und Permission. Die Permission selbst linkt auch wieder auf FennecUser und WebuserData. Da bin ich mir gerade nicht sicher ob diese Mappings so korrekt sind. Würde ich morgen mit Markus gerne nochmal diskutieren.